atest/acceptance/multiple_browsers_executable_path.robot

```diff
@@ -10,7 +10,7 @@ Documentation     Creating test which would work on all browser is not possible.
 
 *** Test Cases ***
 Chrome Browser With executable_path Argument
```

Comment:
https://github.com/robotframework/SeleniumLibrary/pull/1806#discussion_r1057773618 applies here too. Doesn't really matter which PR it's removed in though.